src/pyvesync/vesyncfan.py

```diff
@@ -184,6 +184,8 @@ def build_purifier_dict(self, dev_dict: dict) -> None:
         else:
             self.device_status = 'off'
         self.details['filter_life'] = dev_dict.get('filter_life', 0)
+        self.details['air_quality_value'] = dev_dict.get('air_quality_value',
```

Comment:
I think this belongs under ```if self.air_quality_feature:``` block below since it applies only to those Air Purifiers that support this (Core300S and Core400s support it, Core200s does not)